src/dbnode/testdata/prototest/fixture.go

```diff
@@ -0,0 +1,230 @@
+// Copyright (c) 2019 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package prototest
+
+import (
+	"bytes"
+	"time"
+	"testing"
+
+	"github.com/m3db/m3/src/dbnode/storage/namespace"
+
+	"github.com/jhump/protoreflect/dynamic"
+	"github.com/stretchr/testify/require"
+	"github.com/jhump/protoreflect/desc"
+	"os"
+	"fmt"
+)
+
+type TestMessage struct {
+	timestamp  time.Time
+	latitude   float64
+	longitude  float64
+	epoch      int64
+	deliveryID []byte
+	attributes map[string]string
+}
+
+func NewSchemaHistory(importPath string) namespace.SchemaHistory {
+	srcRoot := os.Getenv("SRC_ROOT")
+	curDir := os.Getenv("PWD")
+	fmt.Printf("SRC_ROOT=%s, PWD=%s, importPath=%s\n", srcRoot, curDir, importPath)
```

Comment:
Do you still need this `fmt.Printf`?  Probably not necessary, or perhaps use a `logger` instead?